File: Service_eHospital_DongNai_A_Config/Implement/Sys_AppPrivateSettingPermissionsService.cs
namespace Service_eHospital_DongNai_A_Config.Implement
{
    public class Sys_AppPrivateSettingPermissionsService : BaseService<Sys_AppPrivateSettingPermissions, ISys_AppPrivateSettingPermissionsRepository>
    , ISys_AppPrivateSettingPermissionsService
    {
    private readonly ISys_AppPrivateSettingPermissionsRepository _Sys_AppPrivateSettingPermissionsRepository;
    public Sys_AppPrivateSettingPermissionsService(ISys_AppPrivateSettingPermissionsRepository Sys_AppPrivateSettingPermissionsRepository) : base(Sys_AppPrivateSettingPermissionsRepository)
    {
    _Sys_AppPrivateSettingPermissionsRepository = Sys_AppPrivateSettingPermissionsRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A_Config/Sys_AppPrivateSettingPermissions.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_AppPrivateSettingPermissions } from './Sys_AppPrivateSettingPermissions.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_AppPrivateSettingPermissionsService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'SettingPermission_Id', 'Setting_Id', 'User_Id', 'Group_Id', 'EditRight', 'DeleteRight'];

    List: Sys_AppPrivateSettingPermissions[] | undefined;
    ListFilter: Sys_AppPrivateSettingPermissions[] | undefined;
    FormData!: Sys_AppPrivateSettingPermissions;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_AppPrivateSettingPermissions";
    }
}

